src/redis_index.c

```diff
@@ -245,7 +248,9 @@ static InvertedIndex *openIndexKeysDict(RedisSearchCtx *ctx, RedisModuleString *
   }
   kdv = rm_calloc(1, sizeof(*kdv));
   kdv->dtor = InvertedIndex_Free;
-  kdv->p = NewInvertedIndex(ctx->spec->flags, 1);
+  size_t sz = 0;
+  kdv->p = NewInvertedIndex(ctx->spec->flags, 1, &sz);
+  ctx->spec->stats.invertedSize += sz;
```

Comment:
Yes. We can avoid it because NewInvertedIndex add the inverted index size to the in/out parameter. Change done in [247a553](https://github.com/RediSearch/RediSearch/pull/4218/commits/247a553f29c32f6495db77c24e56c45a2dd16887)